Gemfile

```diff
@@ -8,11 +8,17 @@ gem 'danger', '~> 8.6'
 gem 'danger-rubocop', '~> 0.10'
 gem 'dotenv'
 gem 'fastlane', '~> 2.174'
+gem 'fastlane-plugin-appcenter', '~> 1.8'
+gem 'fastlane-plugin-sentry'
+# This comment avoids typing to switch to a development version for testing.
+# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 5.5'
```

Comment:
Since I was touching the Fastlane plugins, I took the time to merge `fastlane/Pluginfile` into here.